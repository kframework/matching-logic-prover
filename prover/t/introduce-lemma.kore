claim \equals(1, 1)
strategy introduce-lemma(H: \equals(1,1), by: reflexivity)
       . apply(H, fail)
