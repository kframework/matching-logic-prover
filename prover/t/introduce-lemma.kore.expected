<prover>
  <exit-code>
    0
  </exit-code>
  <goals>
    <goal>
      id: ., parent: .
      <claim>
        \and ( .Patterns )
      </claim>
      <k>
        .
      </k>n<expected>
        .
      </expected>
      <local-context>
        .LocalDeclCellSet
      </local-context>
    <trace>
      .
    </trace>
  </goals>
  <declarations>
    <declaration>
      axiom ax0 : \equals ( 1 , 1 )
    </declaration>
  </declarations>
</prover>
